src/website/app/pages/Styling/styling.md

```diff
@@ -0,0 +1,100 @@
+# Styling
+
+Mineral UI is built on a design system with styles ready to go out of the box.  We realize however that there will be cases when you need to customize styles for you own unique needs.  Below are some different techniques for customizing Mineral UI styles across all levels of your application.
+
+## Customization Techniques
+
+### Themes
+
+Theming is a core concept in Mineral UI.  It is powerful feature capable of providing style customizations from the application level to the component level and should generally be your first choice when looking to make style customizations. See the [theming page](/theming) for more details.
+
+### createStyledComponent
+
+This function is used to create a new styled component based on another component or DOM element.
+
+```jsx
+import { createStyledComponent } from 'mineral-ui/styles';
+import Button from 'mineral-ui/Button';
+
+const myButton = createStyledComponent(Button, {
+  outline: '3px dashed tomato'
+});
+```
+
+See the [API documentation](#styling-api) and more examples below.
+
+### Glamorous API
+
+Mineral UI uses [Glamorous](https://github.com/paypal/glamorous/) for styling components.  Glamorous provides an additional styling mechanism, the `css` prop.  The `css` prop functions similar to the standard React `style` prop, except that styles are applied via a generated CSS class rather than via inline styles.
+
+```jsx
+<Button css={{ outline: '3px dashed tomato' }} />
+```
+
+### CSS classes
+
+Additional CSS classes can be applied to components using the standard React `className` prop.
+
+```jsx
+<Button className="myButton" />
```

Comment:
If they do this, I think they’d be unable to use Glamorous on that component. Maybe not a big deal, but something like this might be better:

```jsx
<Button className={`myButton` ${className}`} />
```

Reply:
I thought of this and tested it in the live-editor on our website and it added an additional class.  I'll do another test elsewhere and update if needed.

Reply:
As discussed, no change needed.